brand/views/res_brand.xml

```diff
@@ -111,15 +108,13 @@
                 <field name="id" />
                 <field name="color" />
                 <field name="display_name" />
-                <field name="title" />
```

Comment:
Why removing some fields ?

Reply:
The title field and object are both removed in v19.